src/main/java/duke/Parser.java

```diff
@@ -110,45 +112,41 @@ private Command handleListInput(String listInput) throws BadInputException,
         switch (keyword[0]) {
         //Commands which are single words.
         case "list":
-            command = new Command(CommandType.LIST);
+            command = new ListCommand(CommandType.LIST);
             break;
         case "bye":
             command = new Command(CommandType.BYE);
             break;
 
         //Commands which require numerical input.
         case "done":
-            command = new NumCommand(CommandType.DONE, Integer.parseInt(keyword[1]));
+            command = new DoneCommand(CommandType.DONE, Integer.parseInt(keyword[1]));
             break;
-        case "delete": {
-            command = new NumCommand(CommandType.DELETE, Integer.parseInt(keyword[1]));
+        case "delete":
+            command = new DeleteCommand(CommandType.DELETE, Integer.parseInt(keyword[1]));
             break;
-        }
 
         //Commands which require string input.
         case "todo":
             String[] todoTemp = addTodo(keyword[1]);
             command = new AddCommand(CommandType.TODO, todoTemp[0],
                     (todoTemp.length > 1) ? todoTemp[1] : "");
             break;
-
-        case "deadline": {
-            String[] temp = addDeadline(keyword[1]);
-            command = new AddCommand(CommandType.DEADLINE, temp[0], temp[1]);
+        case "deadline":
+            String[] deadlineTemp = addDeadline(keyword[1]);
+            command = new AddCommand(CommandType.DEADLINE, deadlineTemp[0], deadlineTemp[1]);
```

Comment:
look into removing commandtype as a whole. As of now i believe it it still there because we need separation between the different addcommands (Whether to add deadline, todo, event and so on..) We can choose to solve this by having a separate class for each add command. Eg. AddDeadline, AddTodo ?

Reply:
i don't even think commandtype is required anymore but i forgot to bring this up with yall. i'm 99% certain its just a relic of me being trigger happy with my enums